mods/ra/mod.yaml

```diff
@@ -403,24 +406,66 @@ ModContent:
 					^Content/ra/temperat.mix:
 						Offset: 499538555
 						Length: 1038859
-		counterstrike: Counterstrike Expansion (English)
+		counterstrike: Counterstrike Expansion Disc (English)
 			IDFiles:
 				README.TXT: 0efe8087383f0b159a9633f891fb5f53c6097cd4
 				SETUP/INSTALL/CSTRIKE.RTP: fae8ba82db71574f6ecd8fb4ff4026fcb65d2adc
 			Install:
 				extract-raw: MAIN.MIX
-					^Content/ra/scores-counterstrike.mix:
-						Offset: 144899491
-						Length: 87483135
-		counterstrike-linux: Counterstrike Expansion (English)
+					^Content/ra/music/2nd_hand.aud:
+						Offset: 209070947
+						Length: 3070092
+					^Content/ra/music/araziod.aud:
+						Offset: 212141039
+						Length: 2941132
+					^Content/ra/music/backstab.aud:
+						Offset: 215082171
+						Length: 3178252
+					^Content/ra/music/chaos2.aud:
+						Offset: 218260423
+						Length: 2860068
+					^Content/ra/music/shut_it.aud:
+						Offset: 221120491
+						Length: 2991979
+					^Content/ra/music/twinmix1.aud:
+						Offset: 224112470
+						Length: 2536972
+					^Content/ra/music/under3.aud:
+						Offset: 226649442
+						Length: 2812788
+					^Content/ra/music/vr2.aud:
+						Offset: 229462230
+						Length: 2920396
```

Comment:
The original version of "Dense" from vanilla RA is cut off at 3:23, while CS has a fixed, full-length version. I'd recommend adding an entry for CS' `dense_r.aud` while you're at it.